app/components/Navbar.test.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import chai, {expect} from 'chai'
+chai.use(require('chai-enzyme')())
+import {mount, shallow} from 'enzyme'
+import {spy, stub} from 'sinon'
+chai.use(require('sinon-chai'))
+
+import NavbarContainer, {Navbar} from './Navbar'
+import {Login} from './Login'
+import {Link} from 'react-router'
+import {createStore} from 'redux'
+
+
+describe('<Navbar />', () => {
+
+  let wrapper
+  beforeEach('render the Navbar wrapper', () => {
+    wrapper = shallow(<Navbar />)
+  })
+
+  it('shows signup and login links when there is no currentUser', () => {
+    expect(wrapper.find('.login-ul').childAt(0)).to.have.html('<li><a>signup</a></li>')
+    expect(wrapper.find('.login-ul').childAt(1)).to.have.html('<li><a>login</a></li>')
+  })
+
```

Comment:
This test is failing with
 
```'TypeError: Cannot read property 'contextTypes' of undefined'``` 
and
```Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in.``` 

Have you gotten that before? Is there a fix?